### ISSUE TITLE
Setup build infrastructure (dependency management, pyproject.toml, ...)

### ISSUE THREAD

**Issue by michaelfruth** [sciunto-org/python-bibtexparser#321]
Setup build infrastructure:

- Use pipenv for development (see https://github.com/streamlit/streamlit/tree/develop/lib as reference)
- Use MakeFile to format, sort, lint, etc..
- Create pyproject.toml

**Comment by MiWeiss** [comment]
Hi @michaelfruth 

Are you still planning on getting back to this issue?

**Comment by michaelfruth** [comment]
Yes, I still *plan* to do it, but I don't have any capacity for this at the moment.

**Comment by MiWeiss** [comment]
stale, re-opened to be grabbed by anyone (including @michaelfruth, if you find the time :-))

**Comment by MiWeiss** [comment]
Fixed in #524
